.circleci/config.yml

```diff
@@ -146,9 +146,14 @@ jobs:
       - setup_vm
       - install_chrome
       - run:
-          name: 'Visual Diff Tests'
-          command: node build-system/pr-check/visual-diff-tests.js
+          name: 'Visual Diff Tests (using << parameters.build_type >> build)'
```

Comment:
On looking at the CI logs, I think it's cleaner to just say `(<< parameters.build_type >>)` instead of `(using << parameters.build_type >> build)`.

Can we trim this name, and the names of the step under `Nomodule Tests`?